src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,876 @@
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/assignments": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Assigns one or more topic partitions to a consumer.",
+                "operationId": "assign",
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "List of topic partitions to assign to the consumer.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/PartitionsForPositions"
+                        }
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "OK"
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    },
+                    "409": {
+                        "description": "Subscription to topics, partitions, and pattern are mutually exclusive."
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer to which you want to assign topic partitions.",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        }
+    },
+    "definitions": {
+        "ApiResponse": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "format": "int32",
+                    "type": "integer"
+                },
+                "type": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                }
+            }
+        },
+        "Consumer": {
+            "title": "Root Type for Consumer",
```

Comment:
Not sure of how we're capitalizing root type in this doc, but consumer has been lower case up to now. 
"Root type for the consumer"
This comment applies to all titles that include 'Root Type...'